bancor_research/bancor_simulator/v3/spec/actions.py

```diff
@@ -342,9 +342,8 @@ def process_withdrawal(
             bnbnt_renounced = bnbnt_rate * bnt_renounced
 
             state.decrease_pooltoken_balance("bnbnt", bnbnt_renounced)
-            state.decrease_protocol_wallet_balance("bnbnt", bnbnt_renounced)
+            state.decrease_protocol_wallet_balance("bnt", bnbnt_renounced)
             state.set_tkn_trading_liquidity(tkn_name, updated_tkn_liquidity)
-            state.decrease_protocol_wallet_balance(f"bn{tkn_name}", pool_token_amt)
```

Comment:
this should be changed to `state.decrease_pooltoken_balance(tkn_name, pool_token_amt)`

Reply:
I tried that, and got a negative balance on bnTKN.

Reply:
Fixed